libpyEM/libpyTypeConverter2.cpp

```diff
@@ -62,7 +63,8 @@ BOOST_PYTHON_MODULE(libpyTypeConverter2)
         .staticmethod("numpy2em")
     ;
 
-    init_numpy();
+    int dummy = 0;
+    dummy = init_numpy();
```

Comment:
Is this necessary?

Reply:
if we follow the old method like we use to do in py2 then we get an error of "ISO C++ forbids declaration of init_numpy with no type [-fpermissive] .

For the time being lets keep it like this :)

Reply:
This code has nothing to do with a function declaration. Calling `init_numpy()` without assigning it's result to anything should be OK. This change here is unnecessary.

Reply:
Dont know why it failed in previous case.   I made the changes you suggested.